Appraisals

```diff
@@ -19,5 +19,5 @@ appraise "rails-6.1" do
 end
 
 appraise "rails-7.0" do
-  gem "rails", "7.0.0"
+  gem "rails", "7.0.1"
```

Comment:
Fixes a permadiff that was introduced here: https://github.com/braintree/pg_ha_migrations/commit/99b3ad41ffe2612217fbfc0863db753402bd03af